.github/workflows/build-deploy-dev.yml

```diff
@@ -125,23 +125,22 @@ jobs:
       - name: Log in to OpenShift
         run: |
           oc login --token=${{ secrets.SA_TOKEN }} --server=https://api.silver.devops.gov.bc.ca:6443
-      # Build the images
-      - name: Build SIMS-API Image
+          if [ $? -ne 0 ]; then
+            echo "Failed to log in to OpenShift. Please check the credentials and connection details."
+            exit 1
+          fi
+       # Build the images in parallel.
+      - name: Build db-migrations, api, workers, queues and web Images in parallel
```

Comment:
`db-migrations` will run first, then rest in parallel right?

Reply:
oh.. since it is deployed we are doing everything in parallel?

Reply:
Yes, you are right

Reply:
The make `oc-something` here, in general, are appended with `--wait` does it not mean that they will be waiting to be executed one after another?

Reply:
No, if the oc-build-db-migrations target in the make command includes --wait in its command, it does not automatically wait for the other make commands to finish. The --wait option typically applies to the specific build process triggered by that particular make target.

Reply:
I got the point when we talked about the commands concatenation with the "&".
Either way, as per the Team discussion, the approach with the GitHub parallel jobs looks great.